articles/quickstart/spa/vuejs/_includes/_centralized_login.md

```diff
@@ -2,475 +2,441 @@
-
-When the **Log In** button is clicked, the user will be redirected to login page.
-
-<%= include('../../_includes/_hosted_login_customization') %>
-
-When the application first starts up, a call to `renewTokens` is made that tries to reinitialize the user's login session, if it is detected that they should already be logged in. This would be the case, for example, if the user logged in and then refreshed the browser window.
+Notice that the configuration file created earlier has been imported and used to initialize the plugin. A custom redirect callback has also been supplied, which redirects the user to a protected route after they have authenticated (this is covered a bit later).
 
-### Add a Callback Component
+## Log in to the App
 
-Using Universal Login means that users are taken away from your application to a login page hosted by Auth0. After they successfully authenticate, they are returned to your application where a client-side session is set for them.
+In this section, you will provide a way for the user to log in or log out, depending on their authentication status.
 
-<%= include('../../_includes/_callback_component') %>
-
-When a user authenticates at the login page and is then redirected back to your application, their authentication information will be contained in a URL hash fragment. The `handleAuthentication` method in the `AuthService` is responsible for processing the hash.
-
-Install `vue-router` to allow callbacks to be routed properly to the `Callback` component:
-
-```bash
-npm install vue-router
-```
-
-Create a component named `Callback` and populate it with a loading indicator. The component should also call `handleAuthentication` from the `AuthService`.
-
-```js
-<!-- src/components/Callback.vue -->
+Open the `views/Home.vue` file and modify the template to include two buttons that enable the user to log in and log out:
 
+```html
 <template>
-  <div>
-    <p>Loading...</p>
+  <div class="home">
+    <img alt="Vue logo" src="../assets/logo.png" />
+    <HelloWorld msg="Welcome to Your Vue.js App" />
+
+    <!-- Check that the SDK client is not currently loading before accessing is methods -->
+    <div v-if="!$auth.loading">
+      <!-- show login when not authenticated -->
+      <button v-if="!$auth.isAuthenticated" @click="login">Log in</button>
+      <!-- show logout when authenticated -->
+      <button v-if="$auth.isAuthenticated" @click="logout">Log out</button>
+    </div>
   </div>
 </template>
+```
 
+Notice that the buttons are wrapped in a directive that makes sure `$auth.loading` is false. This is because most methods on the SDK client are asynchronous, and you must wait until the client has finished loading before trying to access things like the user's authentication state.
+
+Further down the `Home.vue` file, modify the components script and implement the `login` and `logout` methods as follows:
+
+```js
 <script>
+// .. imports removed for brevity
```

Comment:
Not a lot of brevity achieved 😆

Reply:
I thought something was better than nothing 😄